
without this patch the build stops in the test folder on SPARC:

in directory:
/ws/oi-userland/components/scientific/openblas/build/sparcv9/test

OPENBLAS_NUM_THREADS=1 OMP_NUM_THREADS=1 ./dblat1
 Real BLAS Test Program Results
...
 Test of subprogram number  7            DNRM2 

Program received signal SIGBUS: Access to an undefined portion of a memory object.

Backtrace for this error:
/bin/sh: 181992: Abort(coredump)
make[1]: *** [Makefile:52: level1] Abort
make[1]: *** Waiting for unfinished jobs....

reported upstream with: https://github.com/OpenMathLib/OpenBLAS/issues/5552

--- OpenBLAS-0.3.30/test/Makefile.orig	2025-06-19 11:45:39.000000000 +0200
+++ OpenBLAS-0.3.30/test/Makefile	2025-11-27 10:45:42.835735915 +0100
@@ -34,9 +34,6 @@
 ifeq ($(BUILD_SINGLE),1)
 S1=sblat1
 endif
-ifeq ($(BUILD_DOUBLE),1)
-D1=dblat1
-endif
 ifeq ($(BUILD_COMPLEX),1)
 C1=cblat1
 endif
@@ -50,9 +47,6 @@
 ifeq ($(BUILD_SINGLE),1)
 	OPENBLAS_NUM_THREADS=1 OMP_NUM_THREADS=1 ./sblat1
 endif
-ifeq ($(BUILD_DOUBLE),1)
-	OPENBLAS_NUM_THREADS=1 OMP_NUM_THREADS=1 ./dblat1
-endif
 ifeq ($(BUILD_COMPLEX),1)
 	OPENBLAS_NUM_THREADS=1 OMP_NUM_THREADS=1 ./cblat1
 endif
@@ -64,9 +58,6 @@
 ifeq ($(BUILD_SINGLE),1)
 	OMP_NUM_THREADS=2 ./sblat1
 endif
-ifeq ($(BUILD_DOUBLE),1)
-	OMP_NUM_THREADS=2 ./dblat1
-endif
 ifeq ($(BUILD_COMPLEX),1)
 	OMP_NUM_THREADS=2 ./cblat1
 endif
@@ -77,9 +68,6 @@
 ifeq ($(BUILD_SINGLE),1)
 	OPENBLAS_NUM_THREADS=2 ./sblat1
 endif
-ifeq ($(BUILD_DOUBLE),1)
-	OPENBLAS_NUM_THREADS=2 ./dblat1
-endif
 ifeq ($(BUILD_COMPLEX),1)
 	OPENBLAS_NUM_THREADS=2 ./cblat1
 endif
@@ -327,9 +315,6 @@
 endif
 
 ifeq ($(BUILD_DOUBLE),1)
-dblat1 : dblat1.$(SUFFIX) ../$(LIBNAME)
-	$(FC) $(FLDFLAGS) -o dblat1 dblat1.$(SUFFIX) ../$(LIBNAME) $(EXTRALIB) $(CEXTRALIB)
-
 dblat2 : dblat2.$(SUFFIX) ../$(LIBNAME)
 	$(FC) $(FLDFLAGS) -o dblat2 dblat2.$(SUFFIX) ../$(LIBNAME) $(EXTRALIB) $(CEXTRALIB)
 
@@ -385,7 +370,7 @@
 
 clean:
 	@rm -f *.$(SUFFIX) *.$(PSUFFIX) gmon.$(SUFFIX)ut *.SUMM *.cxml *.exe *.pdb *.dwf \
-	sblat1 dblat1 cblat1 zblat1 \
+	sblat1 cblat1 zblat1 \
 	sblat2 dblat2 cblat2 zblat2 \
 	test_sbgemm sblat3 dblat3 cblat3 zblat3 \
 	sblat1p dblat1p cblat1p zblat1p \
